docs/site/Loopback-component-authentication.md

```diff
@@ -159,7 +155,7 @@ export class WhoAmIController {
   @authenticate('basic')
   @get('/whoami')
   whoAmI(): string {
-    return this.userProfile.id;
+    return this.userProfile[securityId];
```

Comment:
Let's add a comment saying `securityId` is `Symbol` to represent the security id of the user, typically the user id.

Reply:
The link will be available when refresh the build of loopback.io

Reply:
Should the link be `https://loopback.io/doc/en/lb4/Security.html`?

Reply:
@raymondfeng I think w/ or w/o `/html` both work, I removed it just to keep the link short.

Like https://loopback.io/doc/en/lb4/Components and https://loopback.io/doc/en/lb4/Components.html